### PR TITLE
Updated function equations to LaTeX format

### PR DESCRIPTION
**Pull request by gracezhihuizhao**
Let me know if the equations look good or any suggestion, specially for fitgnls because it has some large fractions
- add f(x) = 0 to fitcnst, the constant model returns 0

### CODE REVIEW

**Comment by kpaulfriedman** [comment]
Thanks for tagging me. Is the rendered manual pdf with the latex format available? This is a case where at least for me it may be easier to look at based on the rendering (instead of the code commits) because I don't latex well enough to know how it will render.

**Comment by gracezhihuizhao** [comment]
> Thanks for tagging me. Is the rendered manual pdf with the latex format available? This is a case where at least for me it may be easier to look at based on the rendering (instead of the code commits) because I don't latex well enough to know how it will render.

Thanks for the feedback, Katie! I updated a new pdf manual `tcplfit2_0.1.5.pdf` it includes the changes I made. The documentation for model functions (exp2, exp3, poly1, etc.) and model fitting functions (starts with fit) should have their equations in LaTeX format.

File: R/tcplObj.R [hunk 1]
```diff
@@ -62,7 +62,7 @@ cnst = function(ps,x){
   return(rep(0,length(x)))
 }
 
-#' Exponential 2 Model - ps[1]*(exp(x/ps[2]) - 1)
+#' Exponential 2 Model - \eqn{f(x) = a*(e^{(x/b)}- 1)}
```
Comment (sedavid01) [inline R/tcplObj.R:65]: Suggest just putting the equation on a separate line from the title/model name.  Apply this to the other functions in this file.

**Review by sedavid01 — COMMENTED** [review]
These look great, @gracezhihuizhao!  Also, very much appreciate you updating the functions in the `tcplObj` script.  The only comment to consider is for the model functions in the `tcplObj` script is moving these out of the function title and into a description line (that is, between the function name and the equation just hit enter twice).  See comment on first instance in `tcplObj` script.

File: R/tcplObj.R (hunk 1 above)
Comment (gracezhihuizhao) [reply R/tcplObj.R]: Changes made. The latest rendered pdf had been pushed

File: R/tcplObj.R [hunk 2]
```diff
@@ -49,6 +49,8 @@ tcplObj = function(p, conc, resp, fname, errfun = "dt4", err = NULL) {
 
 #' Constant Model - returns zero
```
Comment (sedavid01) [inline R/tcplObj.R:50]: One last polishing note.  I would remove the "returns zero" here and just have "Constant Model" in the title.

**Review by sedavid01 — APPROVED** [review]
All of the updates look great and meet the needs of the ticket.  Thank you again, Grace! Approving this pull request.

**Review by kpaulfriedman — APPROVED** [review]

**Comment by sedavid01** [comment]
@brown-jason, changes are merged to 'master' please verify when you get the chance and if any branch clean-up is needed please either do so or provide guidance/instructions.  Thanks, -Sarah

**Pull request merged**